.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,4 +40,7 @@ or AWS CLI commands to trigger something.
 - [ ] Fix linting errors
 - [ ] Make sure code coverage hasn't dropped
```

Comment:
Is this necessary as we can configure Coveralls to fail when the coverage drops?